stable/mtls-proxy/templates/deployment.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: {{ include "mtls-proxy.fullname" . }}
   labels:
 {{ include "mtls-proxy.labels" . | indent 4 }}
+{{- with .Values.deployment.annotations }}
```

Comment:
just to double check, can you verify that empty annotation here behaves and deploys as expected.

Reply:
ok tested this, looks great, thank!